python/pyarrow/array.pyx

```diff
@@ -329,13 +329,14 @@ def from_pandas_series(object series, object mask=None, timestamps_to_ms=False):
     Parameters
     ----------
     series : pandas.Series or numpy.ndarray
+        timestamps will be auto-converted to UTC since Arrow only stores in this timezone
```

Comment:
It would be better to put this in a `Notes` section, or in the Sphinx documentation